docs/sphinx/reference-encoders.rst

```diff
@@ -443,6 +443,12 @@ General Encoder Functions
 
 ---------------------
 
+.. function:: size_t obs_encoder_get_mixer_index(const obs_encoder_t *encoder)
+
+   :return: The mixer index of the encoder
```

Comment:
Maybe be a bit more explicit, "The mixer index for the audio track which is encoded by the encoder."